src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -11,6 +11,9 @@ public partial class ScrollViewHandler : ViewHandler<IScrollView, UIScrollView>,
 	{
 		const nint ContentTag = 0x845fed;
 
+		//Scroll view not resizing correctly when orientation changes
+		public override bool NeedsContainer => true;
```

Comment:
The build is failing:

`D:\a\_work\1\s\src\Core\src\PublicAPI\net-maccatalyst\PublicAPI.Unshipped.txt(90,1): error RS0050: Symbol 'override Microsoft.Maui.Handlers.ScrollViewHandler.NeedsContainer.get -> bool' is marked as removed but it isn't deleted in source code (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net9.0-maccatalyst18.0]`

Could you review the publicAPI file?

Reply:
@jsuarezruiz , I have updated the code changes.